File: python/compare.py
# Lecture 06 - "Python"
# Program 05

def main():
    x = get_int("What's x? ")
    y = get_int("What's y? ")

    if x < y:
        print("X is less than y")
    elif x > y:
        print("X is greater than y")
    else:
        print("X is equal to y")


def get_int(prompt):
    # Get int from user and return
    while True:
        try:
            result = int(input(prompt))
            return result
        except ValueError:
            print("Expected an 'int', try again!")


if __name__ == '__main__':
    main()

File: c/compare.c
// Compare.c
// Program to take inputs as integers and compare the integes


#include <stdio.h>

int main(void)

{
    int x, y;
    // Taking inputs
    printf("What's x? ");
    scanf("%i", &x);

    printf("What's y? ");
    scanf("%i", &y);

    // Printing inputs
    printf("x:%i, y:%i\n", x, y);

    // Comparing inputs
    if (x < y)
    {
        printf("X is less than Y.\n");
    }
    else if (x > y)
    {
        printf("X is greater than Y.\n");
    }
    else
    {
        printf("X is equal to Y.\n");
    }

    return 0;
}